pyramid_okta/__init__.py

```diff
@@ -1,10 +1,25 @@
-from . import settings
+from pyramid_okta import settings
 from pyramid_okta.security.authentication import OktaAuthenticationPolicy
 from pyramid_okta.security.authorization import OktaAuthorizationPolicy
+from pyramid_okta.request import OktaRequestExtra
+from pyramid_okta.session import OktaSessionFactory
 
 
 def includeme(config):
     settings.API_TOKEN = config.registry.settings.get('okta.api_token', settings.API_TOKEN)
     settings.BASE_URL = config.registry.settings.get('okta.base_url', settings.BASE_URL)
+
+    # set up authn/z policies
     config.set_authentication_policy(OktaAuthenticationPolicy())
-    config.set_authorization_policy(OktaAuthorizationPolicy())
\ No newline at end of file
+    config.set_authorization_policy(OktaAuthorizationPolicy())
```

Comment:
should we be setting this by default?  perhaps a config option -- if you want to include `pyramid_okta` but subclass it vs. set it by default, would that be problematic?